packages/website/components/GlobalStyles.tsx

```diff
@@ -63,6 +63,10 @@ export const GlobalStyles = () => {
           padding: 0;
         }
 
+        img {
```

Comment:
I'm not sure if this style should go to global styles. I'd say it should go to PageContent only

Reply:
when I tried to add this to MdxRenderer, next js asked me to use next/Image
but it has height and width as mandatory props
https://nextjs.org/docs/api-reference/next/image#required-props

wdyt?

Reply:
the height I will have to add it manually

Reply:
Not sure I get it right. If we don't know width and height then we should use a simple img not `next/Image` What exect image we want to import with `next/Image`?

Reply:
the typography page has two images

you can check my change in MdxRenderer.tsx now, I had to add to eslint-disable rules
and one of them is there because next/js warns that I'm using a `img` tag and they recommend using their component

but their component needs width, height, and src
and when I put, for example, `width="100%" height={340}` for the first image
the image get distorted 
`layout="fill"` makes it the size of the screen
and so on

so I decided to use `img` and disable the rules